src/routes/root.tsx

```diff
@@ -0,0 +1,50 @@
+import { Tabs, Anchor } from "@mantine/core";
+import { IconMessage, IconUser, IconHome } from "@tabler/icons-react";
+import { Outlet } from "react-router-dom";
+
+function Root() {
+  return (
+    <>
+      <Tabs defaultValue="gallery">
+        <Tabs.List>
+          <Anchor href="/">
```

Comment:
you need to use the `<Link to="..." />` from `react-router-dom`, otherwise the browser refreshes the app when clicking a link.
that's the default browser behaviour when clicking an anchor tag.
since the routing we want is client-side-routing, we must use the tools we're given by `react-router`.

also, in mantine's documentation there's an example of using the `Tabs` component with `react-router`.